Editor/Deployment/DeploymentSettings.cs

```diff
@@ -421,6 +420,13 @@ public async Task StartDeployment(ConfirmChangesDelegate confirmChanges)
             }
         }
 
+        public async Task DeleteDeployment()
+        {
+            var stackName = _coreApi.GetStackName(_gameName);
+            _coreApi.DeleteStack(CurrentProfile, CurrentRegion, stackName);
+            RefreshCurrentStackInfo();
```

Comment:
It's probably more appropriate to directly call `ClearCurrentStackInfo` in this case?

Reply:
I don't believe it will immediately be deleted, so refreshing to get the delete in progress status